internal/gatewayapi/resource/load.go

```diff
@@ -355,6 +355,19 @@ func loadKubernetesYAMLToResources(input []byte, addMissingResources bool) (*Res
 				Spec: typedSpec.(gwapiv1a3.BackendTLSPolicySpec),
 			}
 			resources.BackendTLSPolicies = append(resources.BackendTLSPolicies, backendTLSPolicy)
+		case KindEnvoyExtensionPolicy:
```

Comment:
can you also remove L49 & L51? they are supported now

Reply:
done